### PR TITLE
Added a Capture function to lua

### PR DESCRIPTION
**Pull request by abcdefg30**
Split from #8416 as requested: https://github.com/OpenRA/OpenRA/pull/8416#discussion_r32369236

Has already one :+1: from https://github.com/OpenRA/OpenRA/pull/8416#issuecomment-113252033.

### CODE REVIEW

File: OpenRA.Mods.Common/Scripting/Properties/CaptureProperties.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		[Desc("Captures the target actor.")]
+		public void Capture(Actor target)
+		{
+			var normalCapturable = target.Info.Traits.GetOrDefault<CapturableInfo>();
+			var externalCapturable = target.Info.Traits.GetOrDefault<ExternalCapturableInfo>();
+
+			if (normalInfo != null && normalCapturable != null && normalInfo.CaptureTypes.Contains(normalCapturable.Type))
+				Self.QueueActivity(new CaptureActor(Self, target));
+			else if (externalInfo != null && externalCapturable != null && externalInfo.CaptureTypes.Contains(externalCapturable.Type))
+				Self.QueueActivity(new ExternalCaptureActor(Self, Target.FromActor(target)));
```
Comment (penev92) [inline OpenRA.Mods.Common/Scripting/Properties/CaptureProperties.cs]: How about an `else throw new InvalidOperationException("Actor X cannot capture")` or something, so when someone screws up it doesn't fail silently.

**Comment by abcdefg30** [comment]
Added `else throw new LuaException("Actor '{0}' cannot capture actor '{1}'!".F(Self, target));`.

**Comment by penev92** [comment]
Thanks.
Looks good to me now :+1:

**Comment by penev92** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/4d1f5bc59a646c0134e6c1a6d8954a3a14ac5162...074e440714b2aad490d09f8cdf40f9d6dc8f0625)

**Pull request merged**